tier4_system_msgs/msg/EmergencyGoalsClearCommand.msg

```diff
@@ -0,0 +1,3 @@
+builtin_interfaces/Time stamp
+bool command false
```

Comment:
What is this used for?

Reply:
It's unnecessary filed, so I'll delete it.